projects/angular/src/layout/vertical-nav/vertical-nav-group.spec.ts

```diff
@@ -232,6 +232,12 @@ export default function (): void {
         fixture.detectChanges();
         expect(button.getAttribute('aria-expanded')).toBe('true');
       });
+
+      it('should set the aria-label attribute on the expand/collapse button', () => {
+        const button: HTMLButtonElement = compiled.querySelector('.nav-group-trigger');
+        expect(button.hasAttribute('aria-label')).toBe(true);
+        expect(button.getAttribute('aria-label')).toContain('Link');
```

Comment:
This is not correct.
1. The button has visible text that says "Group". Why do we need an aria-label on a button that has visible text?
2. There is a link in the group with "Link" as the visible text (and accessible name). Accessible names must be unique.

Reply:
You're right, It seems that there is an issue with how the component is implemented in storybook and not the component itself.  There is a related JIRA to track this: [CDE-1390](https://jira.eng.vmware.com/browse/CDE-1390)
Closing this PR.